Week1/Section2/Exercise7/Source7.cpp

```diff
@@ -0,0 +1,95 @@
+/*
+Exercise 7 - Rock, Paper, Scissors
+Write a simple rock, paper, scissors game.
+Ask the user for their guess on the result, then tell them whether they guessed right or wrong.
+
+Example output
+Welcome to Rock, Paper, Scissors!
+
+1 - Rock
+2 - Paper
+3 - Scissors
+Choose your weapon: 2
+
+Your choice: Paper
+Computer�s choice: Rock
+
+You win� big deal!
+*/
+
+#include <iostream>
+
+enum options {
+	rock = 1, paper, scissors
+};
+
+std::string toText(int a) {
```

Comment:
You may be better off passing the parameter as an `options` rather than an `int`. This is because you are not returning the int as text, but rather the enum value.

Reply:
I made this change but it gives me a casting error, I fixed it but I don't know if it was the right way.